person.rb

```diff
@@ -10,6 +10,12 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rentals.person = self
```

Comment:
- [ ] same here, Please make this method to create a new rental, not just adding rentals to an array. The book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
[Hint] See this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more info.

Reply:
I have also make the method to create a new rental

Reply:
Kindly update the logic on line 18 here to `person.rentals = self`.

A person in the case would serve as a user of the system. And it is better to make references beginning from the user even if it's a many-to-many relationship.